examples/evaluations/classification/classification_metrics_demo.ipynb

```diff
@@ -153,23 +153,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 5,
    "id": "33e61ded-b56f-42f3-897a-0df80d03b626",
    "metadata": {
     "tags": []
    },
    "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-------------\n",
+      "FalseNegativeRateParity\n",
+      "Sklearn Confusion matrix, CM1:  [[2519 1032]\n",
```

Comment:
Can we re-run this after removing the printing of the confusion matrix?

Reply:
Done